### PR TITLE
[UPD] .travis.yml definition

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
fixes travis builds.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,20 +17,14 @@ addons:
 
 env:
   global:
-  - VERSION="11.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - VERSION="11.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
   - TRANSIFEX_USER='transbot@odoo-community.org'
```
Comment (pedrobaeza) [inline .travis.yml]: You can remove this part

File: .travis.yml [hunk 2]
```diff
@@ -19,20 +17,12 @@ addons:
 
 env:
   global:
-  - VERSION="11.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
-  - secure: cnskjzYSLkdeEoUDIhwkwqWsNKIj36JyoA/H7ByNLBphAm9Xrs/ZE8Y5AHxsXp8zQjIGnzw4fpEEO8oakmaueFTaAKlzWS2KH8NWxTjNHkIv/+TFwYsayWNG6O39021wSMQnTRDVERNlBdTwjCTBne81a50NqCeUIrtgcwycwzU=
```
Comment (LoisRForgeFlow) [inline .travis.yml:24]: @pedrobaeza is this still needed?

File: .travis.yml (hunk 1 above)
Comment (LoisRForgeFlow) [reply .travis.yml]: done

File: .travis.yml (hunk 2 above)
Comment (pedrobaeza) [reply .travis.yml:24]: No

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**